### PR TITLE
Update README.md to include other racket channels

### PR DESCRIPTION
**Pull request by spdegabrielle**
updated PR with no changes to build.md

### CODE REVIEW

**Comment by spdegabrielle** [comment]
@sorawee no changes to build.md as requested.  https://github.com/racket/racket/pull/4123#issuecomment-1017971319

File: README.md [hunk 1]
```diff
@@ -30,20 +30,25 @@ For information on building Racket from this repository, see the
...
 
 By making a contribution, you are agreeing that your contribution is
 licensed under the LGPLv3, Apache 2.0, and MIT licenses. Those
 licenses are available in this repository in the files
 racket/src/LICENSE-LGPL.txt, racket/src/LICENSE-APACHE.txt, and
 racket/src/LICENSE-MIT.txt.
 
-See the [Racket Build Guide](build.md) for more guidance on
-contributing.
+See the [Building, Distributing, and Contributing to Racket](https://docs.racket-lang.org/racket-build-guide/index.html)
```
Comment (sorawee) [inline README.md:47]: If the goal is to solve the "inundating information" issue, I don't think listing two links here (without making it clear that they are the same) will help.

What about this? For now, change this line to:

```
See the [Building, Distributing, and Contributing to Racket] (build.md) for more guidance on contributing.
```

Later, we could change https://github.com/racket/racket/blob/master/pkgs/racket-build-guide/racket-build-guide.scrbl to say:

```
The online HTML version can be accessed at https://docs.racket-lang.org/racket-build-guide/
```

Comment (spdegabrielle) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -30,20 +30,25 @@ For information on building Racket from this repository, see the
...
 ------------
 
-Contribute to Racket by [submitting a pull request](https://github.com/racket/racket), joining the
-[development mailing list](https://lists.racket-lang.org), or visiting
-the [IRC](https://racket-lang.org/community.html) or [Slack](https://racket-slack.herokuapp.com/) channels.
+Contribute to Racket by
+* [submitting a pull request](https://github.com/racket/racket), 
+* visiting the Racket forums 
+  * [Racket Discussions Discourse](https://racket.discourse.group/)
+  * [Racket Discord}(https://discord.gg/6Zq8sH5) #internals
```
Comment (jryans) [inline README.md:37]:
```suggestion
  * [Racket Discord](https://discord.gg/6Zq8sH5) #internals
```

File: README.md [hunk 3]
```diff
@@ -30,20 +30,25 @@ For information on building Racket from this repository, see the
...
 
 By making a contribution, you are agreeing that your contribution is
 licensed under the LGPLv3, Apache 2.0, and MIT licenses. Those
 licenses are available in this repository in the files
 racket/src/LICENSE-LGPL.txt, racket/src/LICENSE-APACHE.txt, and
 racket/src/LICENSE-MIT.txt.
 
-See the [Racket Build Guide](build.md) for more guidance on
-contributing.
+See the [Building, Distributing, and Contributing to Racket] (build.md)
```
Comment (jryans) [inline README.md:47]:
```suggestion
See the [Building, Distributing, and Contributing to Racket](build.md) 
```

**Review by jryans — COMMENTED** [review]
A few suggestions to fix some Markdown syntax errors.

**Pull request merged**